wdio.conf.js

```diff
@@ -159,6 +160,7 @@ exports.config = {
             './src/steps/given.js',
             './src/steps/then.js',
             './src/steps/when.js',
+            './src/support/hooks.js',
```

Comment:
Why do you require hooks in here?

Reply:
Sorry unintended change

Reply:
I have pushed change onto same branch but it doesn't reflect on this merge request, Hence I have created duplicate MR